lib/account/pages/account_page.dart

```diff
@@ -15,11 +20,68 @@ class _AccountPageState extends State<AccountPage> {
     supabaseClient.auth.signOut();
   }
 
+  void showDesignDialog(BuildContext context) {
+    showDialog(
+      context: context,
+      builder: (BuildContext context) => StatefulBuilder(
+        builder: (context, setState) => AlertDialog(
+          content: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: List.generate(
+              ThemeMode.values.length,
+              (index) => RadioListTile(
+                  title: Text(
+                    ThemeMode.values[index].getName(context),
+                  ),
+                  value: ThemeMode.values[index],
+                  groupValue: themeManager.currentThemeMode,
+                  onChanged: (ThemeMode? value) {
+                    setState(() {
+                      themeManager.setTheme(value);
+                    });
+                  }),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  void changeTheme(ThemeMode? themeMode) {
+    if (themeMode == null) return;
+  }
+
+  void showLanguageDialog(BuildContext context) {
+    showDialog(
+      context: context,
+      builder: (BuildContext context) => StatefulBuilder(
+        builder: (context, setState) => AlertDialog(
+          content: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: List.generate(
+              localeManager.supportedLocales.length,
+              (index) => RadioListTile(
+                title: Text(localeManager.supportedLocales.map((e) => e.languageName).toList()[index]),
+                value: localeManager.supportedLocales[index],
+                groupValue: localeManager.currentLocale,
+                onChanged: (Locale? value) {
+                  setState(() {
+                    localeManager.setCurrentLocale(value);
+                  });
```

Comment:
```suggestion
                  });
                  Navigator.of(context).pop();
```
or add a close button to the dialog.